search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -15,21 +15,15 @@
 // The default score assigned to countries with no search results.
 const POSITIVE_COLOR = 'green';
 const NEGATIVE_COLOR = 'red';
-
```

Comment:
Add this newline back. Seems good to have a gap between the constants and the functions.